addons/website_crm_partner_assign/models/crm_lead.py

```diff
@@ -30,6 +30,15 @@ def _merge_data(self, fields):
         fields += ['partner_latitude', 'partner_longitude', 'partner_assigned_id', 'date_partner_assign']
         return super(CrmLead, self)._merge_data(fields)
 
+    def update_contact_details_from_portal(self, values):
```

Comment:
Could you put it below compute and main action methods ? It is currently locate a bit randomly in the file, it makes file hard to understand ...

Reply:
My bad, done :+1: Thanks!